docs/pages/docs/providers/node.md

```diff
@@ -42,6 +43,8 @@ All dependencies found in `packages.json` are installed with either NPM, Yarn, o
 
 The build script found in `package.json` if it exists or if its an NX Monorepo `(npm|pnpm|yarn|bun) run build <NxAppName> --configuration=production`.
 
+Or, if it's a Turborepo monorepo (detected if `turbo.json` exists), the `build` pipeline will be called, if it exists; otherwise, the `build` script of the package referenced by `NIXPACKS_TURBO_APP_NAME` will be called, if the variable is provided; otherwise, it will fall back to the build script found in `package.json`.
```

Comment:
```suggestion
Or, if it's a Turborepo monorepo (detected if `turbo.json` exists), the `build` pipeline will be called (if it exists). Otherwise, the `build` script of the `package.json` referenced by `NIXPACKS_TURBO_APP_NAME` will be called, if `NIXPACKS_TURBO_APP_NAME` is provided and valid. Otherwise, it will fall back to the build script found in `package.json` at the monorepos root.
```

Reply:
It doesn't actually check the validity of `NIXPACKS_TURBO_APP_NAME`. Should it? I don't think that the NX code does.

Reply:
It should. I'll implement support for Nx and validating the name in another PR.